src/modules/sensors/vehicle_imu/VehicleIMU.cpp

```diff
@@ -529,7 +529,7 @@ bool VehicleIMU::Publish()
 {
 	bool updated = false;
 
-	vehicle_imu_s imu;
+	vehicle_imu_s imu {};
```

Comment:
What was the issue here? In a lot of cases it might actually be better to catch the particular field that isn't being populated.

Reply:
I didn't understand any of it. Feel free to improve it.

Reply:
I mean did valgrind tell you the particular uninitialized field?

Reply:
No, it didn't.

Easy to reproduce:

```
make px4_sitl gazebo_iris_valgrind
```